### PR TITLE
updates for homepage design and add help us grow section

### PR DESCRIPTION
**Pull request by LavMatt**
This PR addresses two issues #609 and #637, making both design changes and additions to the homepage of find-moj-data.

The resulting changes can be seen below:
image

### CODE REVIEW

File: locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -359,14 +347,53 @@ msgstr "Filter this page"
...
+msgid "Discover data from across the Ministry of Justice"
+msgstr "Discover data from across the Ministry of Justice"
+
+#: templates/home.html:12
+#, fuzzy
+#| msgid ""
+#| "Find MoJ data shows you what data exists and how to access it. <a "
+#| "style=\"color: white;\" href=\"https://user-guide.find-moj-data.service."
+#| "justice.gov.uk/\">Learn more</a>"
+msgid ""
```
[the remark this answers is not in the document]
Comment (LavMatt) [reply locale/en/LC_MESSAGES/django.po:360]: Yeah i think i agree

File: locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -359,14 +347,53 @@ msgstr "Filter this page"
...
+"Find MoJ data shows you what data exists and how to access it. <a style=\\"
+msgstr ""
+"Find MoJ data shows you what data exists and how to access it. <a "
+"style=\"color: white;\" href=\"https://user-guide.find-moj-data.service."
+"justice.gov.uk/\">Learn more</a>"
```
[the remark this answers is not in the document]
Comment (LavMatt) [reply locale/en/LC_MESSAGES/django.po:358]: is as per Mark's design. Is the preceding text not giving the link some context?

[the remark this answers is not in the document]
Comment (LavMatt) [reply locale/en/LC_MESSAGES/django.po:358]: actually you are right. Just tried with voiceover on and it only reads out the link

[the remark this answers is not in the document]
Comment (LavMatt) [reply locale/en/LC_MESSAGES/django.po]: i've changed

[the remark this answers is not in the document]
Comment (LavMatt) [reply locale/en/LC_MESSAGES/django.po:358]: Mark suggested aria-label which i've added and has worked ok

**Review by murdo-moj — APPROVED** [review]

**Pull request merged**